### PR TITLE
CakePHP

### PR DESCRIPTION
**Pull request by Haradwaith**
NOTE:
- Please fill out the template below 

## Did you read the instructions in the [README](https://github.com/the-ethan-hunt/first-timers-guide/blob/master/README.md)?

> Yes

## Have you read the instructions in [CONTRIBUTING](https://github.com/the-ethan-hunt/first-timers-guide/blob/master/CONTRIBUTING.md)?

> Yes

## Have you written this article by yourself? Have you taken inspiration from anywhere and then written it in your words?

> Thank you from France to create this project ! 
We participate in the context of classes to learn about how to contribut !
Have a nice day!

### CODE REVIEW

**Comment by the-ethan-hunt** [comment]
Nice topic to pick up!
But the article isn't up to mark. You need to explain:
- What CakePHP is
- Instructions on implementing it
- Its pros and cons

Do this and the PR is ready to be merged

File: guides/Framework/CakePHP [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# CakePhp
+
+## What is it ? 
+
+This is an open source php Framework based on MVC pattern. Until MIT licence.
```
Comment (penguinpet) [inline guides/Framework/CakePHP]: 'Cake' is an open source, sounds better than this is an open source

Comment (the-ethan-hunt) [reply guides/Framework/CakePHP]: @penguinpet , quoting from the `CONTRIBUTING.md`:
>The maintainer will review your PR in an span of 2 days. Do not review other contributor's PR until said so by the maintainer himself.
Please don't review PRs until said so by the maintainer(me)

**Comment by the-ethan-hunt** [comment]
Closing this PR

**Pull request closed without merging**